src/localization/messages_Hi_IN.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Translated default messages for the jQuery validation plugin.
+ * Locale: Hi-IN (Hindi; हिंदी)
```

Comment:
```suggestion
 * Locale: Hindi; हिंदी
```

Reply:
@bytestream Updated the suggested changes